code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -382,7 +382,7 @@ obj/effect/sweatsplash/proc/splash()
 	switch(A.stage)
 		if(4, 5)
 			if(burnheal)
-				M.adjustFireLoss(-1 * power) //we don't care about bodypart type here, because this is rolling back time or some shit
+				M.heal_overall_damage(0, 1) //no required_status checks here, this does all bodyparts equally
```

Comment:
Would be nicer / more obvious what this was doing exactly with
M.heal_overall_damage(burn=1) //no required_status checks here, this does all bodyparts equally

Reply:
adjustfireloss automatically applies a required_status of BODYPART_ORGANIC to bodyparts. heal overall damage doesnt, so it heals robotic limbs